Update UG to reflect multiple arguments behavior

tanqinyong opened the pull request:
For notes and levels, multiple prefixes are allowed in the 'add' and 'edit' command.

However, only the last prefix is considered and taken as input.

We reflect this behavior in the UG accordingly so users are not confused.

This clarifies/fixes #146 and #189

tanqinyong committed:
Update UG to reflect multiple arguments behavior

For notes and levels, multiple prefixes are allowed.

However, only the last prefix is considered and taken as input.

We reflect this behavior in the UG accordingly so users are not
confused.

This fixes #146 #189

tanqinyong committed:
Minor changes

Add minor changes and update edit command to reflect
the behaviour too.

tanqinyong committed:
Merge branch 'master' into UG-multiple-arguments-fix

Darylgolden reviewed (approved):
LGTM! Do you think the inconsistent behavior should be classified as a feature flaw and reflected as a planned enhancement?

tanqinyong commented:
add in ... for appointments and update DG

tanqinyong committed:
Update UG to include ... for appointments

Updated the UG to include ... for appointments as it
was forgotten.

Also updated the command summary at the bottom

tanqinyong committed:
Update the DG to include inconsistent prefix behavior

Replace #8 planned enhancement accordingly.

tanqinyong committed:
Merge branch 'UG-multiple-arguments-fix' of https://github.com/tanqinyong/tp into UG-multiple-arguments-fix

# Conflicts:
#	docs/UserGuide.md

tanqinyong committed:
Merge branch 'master' into UG-multiple-arguments-fix
